src/actions/editThought.ts

```diff
@@ -134,10 +134,12 @@ const editThought = (state: State, { cursorOffset, force, oldValue, newValue, pa
     [oldKey]: newOldLexeme,
     [newKey]: lexemeNew,
   }
-
-  const sortPreference = getSortPreference(state, editedThought.parentId)
+  const isNote = parentOfEditedThought.value === '=note'
+  const parentThoughtId = isNote
+    ? (getThoughtById(state, parentOfEditedThought.parentId)?.parentId ?? editedThought.parentId)
```

Comment:
Generally I prefer `rootedParentOf(parentOf(path))` to access grandparents rather than using `getThoughtById`.